WordPress/src/main/java/org/wordpress/android/ui/posts/adapters/PostsListAdapter.java

```diff
@@ -368,6 +373,19 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
         listPopup.show();
     }
 
+    private void updatePostUploadProgressBar(ProgressBar view, PostModel post) {
+        if (UploadService.isPostUploadingOrQueued(post)) {
```

Comment:
Should we change the [indeterminateBehavior](https://developer.android.com/reference/android/widget/ProgressBar.html#attr_android:indeterminateBehavior) to `cycle` when the post is queued, to make it visually different than the upload progress?